### PR TITLE
fix(steam-deb): Enable 32 bit support by default

### LINKED ISSUE #838: `steam-deb` uses 32bit deb
**Describe the bug**
Package `steam-deb` uses arch `i386` (`86x32`)

**To Reproduce**
`pacstall -I steam-deb`

**Expected behavior**
Should also have `x64` arch as it is the most used one

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Debian
 - Bullseye `11`

### PR DESCRIPTION
**Pull request by WRM-42**
Fixes #838

### CODE REVIEW

**Comment by wizard-28** [comment]
@Henryws handle this. The CI fails due to `i386` errors.

File: packages/steam-deb/steam-deb.pacscript [hunk 1]
```diff
@@ -2,8 +2,14 @@ name="steam-deb"
...
+breaks="${gives}-bin ${gives}-app"
+replace="${gives}"
 url="https://repo.${gives}powered.com/${gives}/archive/stable/${gives}_${version}_i386.deb"
 description="Steam is a video game digital distribution service by Valve"
 hash="cd28218ebd64c74b5c1d654d5df467be18f5e88db907eb86ac620b51f445d659"
 maintainer="WRM-42 <y8bsbahy@anonaddy.me>"
+repology=("project: ${gives}")
+
+prepare() {
+    sudo dpkg --add-architecture i386
```
Comment (Henryws) [inline packages/steam-deb/steam-deb.pacscript:14]:
```suggestion
    sudo dpkg --add-architecture i386
    sudo apt-get update
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by wizard-28** [comment]
@Henryws

**Comment by WRM-42** [comment]
I did a test on a virtual machine and it works well

https://bin.snopyta.org/?8096c735389a0c0c#38hMMR5ef3hx4mVkjVrbToenTtjLdjDCmYXMDNzHPAsW

**Comment by Henryws** [comment]
> I did a test on a virtual machine and it works well
> 
> [bin.snopyta.org/?8096c735389a0c0c#38hMMR5ef3hx4mVkjVrbToenTtjLdjDCmYXMDNzHPAsW](https://bin.snopyta.org/?8096c735389a0c0c#38hMMR5ef3hx4mVkjVrbToenTtjLdjDCmYXMDNzHPAsW)

Yup, it's an issue with the CI

**Pull request merged**